src/PrestaShopBundle/ApiPlatform/Resources/Hook.php

```diff
@@ -84,17 +87,20 @@
             requirements: ['id' => '\d+'],
             exceptionToStatus: [HookNotFoundException::class => 404],
             provider: QueryProvider::class,
-            extraProperties: ['query' => GetHook::class]
+            extraProperties: [
+                'query' => GetHook::class,
+                'scopes' => ['hook_read'],
+            ]
         ),
     ],
 )]
 class Hook
 {
     #[ApiProperty(identifier: true)]
-    #[Groups(['hook-status:read'])]
+    #[Groups(['hook-status_read'])]
     public int $id;
 
-    #[Groups(['hook-status:read'])]
+    #[Groups(['hook-status_read'])]
```

Comment:
```suggestion
    #[Groups(['hook_status_read'])]
```

Reply:
Unless we can completely remove those (as explained in previous comment) which would be even better